source/broadband/webpa_adapter.c

```diff
@@ -284,14 +284,17 @@ void processRequest(char *reqPayload,char *transactionId, char **resPayload)
                                 resObj->u.paramRes->params = NULL;
                                 
                                 WalInfo("Request:> newCid: %s oldCid: %s syncCmc: %s\n",reqObj->u.testSetReq->newCid, reqObj->u.testSetReq->oldCid, reqObj->u.testSetReq->syncCmc);
-                                // Get CMC from device database
+                                if( dbCMC != NULL && dbCID != NULL)
```

Comment:
Before the actual get for dbcmc and dbcid, value will be NULL. See initialization in Line 53, 54